DeAceroDLabs/marketplace-frontend
# Feature/dmp 518 Corrections

## What is done

[[DMP-518] Corrections](https://deacero.atlassian.net/browse/DMP-518)

## How is done

- Correted `SearchBar` all the placeholder fields on`Sign up`.
- Change style to form-description so that is grey as figma.
- Correted everything else according to figma and excel:
https://docs.google.com/spreadsheets/d/1AExCV1P6z0JRPtWGQOnlA7AR9_kr9NXHoeJ4jYub0mE/edit#gid=0

## How to test

1. Run the project via `npm start`
2. signup
3. You should see everything corrected.
